Update conclusion.Rmd (#56)
danieldc98/NYCneighborhoodInternetAdoption: conclusion.Rmd

# Conclusion

> We have explored the relation of internet availability and various socioeconomic features of neighborhoods throughout the five boroughs of New York City. In doing so, we discovered some trends in availability and adoption of internet that indicate the shortcomings of Mayor De Blasio's "Internet Master Plan". 

Reviewer: ```suggestion
We have explored the relation of internet availability and various socioeconomic features of neighborhoods throughout the five boroughs of New York City. In doing so, we discovered trends in availability and adoption of internet that suggest a gap between Mayor De Blasio's "Internet Master Plan" and reality. 
```

> Looking at internet adoption and availability by boroughs, we found that the best internet coverage is in Manhattan. Staten Island and Brooklyn have mid to high level internet adoption, while Queens comes in slightly higher than the Bronx, but both at the bottom of the list. We came to this conclusion using data on internet adoption and availability, internet speed availability, public hot spot availability and mobile internet dependency. 

Reviewer: ```suggestion
Looking at internet adoption and availability by borough, we found that the best internet coverage is in Manhattan. Staten Island and Brooklyn have mid to high level internet adoption, while Queens comes in slightly higher than the Bronx, but both at the bottom of the list. These conclusions are based on internet adoption, availability, speed, public accessibility, and mobile dependency data. 
```

> One of the most concerning findings is the relation of mean income and internet availability and adoption. Referring to the graph above we can see that overall, a higher income in a neighborhood correlates with better internet availability and adoption. This trend is further emphasized when looking at race, housing and other demographic features. First, it is important to note the there are several demographic features that are closely correlated in society: number of children is correlated to poverty rates, rental prices and home values are correlated to income and often times, and neighborhoods in cities are often predominantly made up of a single race/ethnicity. With that being said, we found a correlation between these demographic features and internet adoption and availability.

Reviewer: ```suggestion
One of the most concerning findings is the relation of mean income and internet availability and adoption. Referring to the graph above we can see that overall, a higher income in a neighborhood correlates with better internet availability and adoption. This trend is further emphasized when looking at race, housing and other demographic features. First, it is important to note the there are several demographic features that are closely correlated in society: number of children is correlated to poverty rates, rental prices and home values are correlated to income and often, neighborhoods in cities are predominantly made up of a single race/ethnicity. With that being said, we found a correlation between these demographic features and internet adoption and availability.
```

> One of the most concerning findings is the relation of mean income and internet availability and adoption. Referring to the graph above we can see that overall, a higher income in a neighborhood correlates with better internet availability and adoption. This trend is further emphasized when looking at race, housing and other demographic features. First, it is important to note the there are several demographic features that are closely correlated in society: number of children is correlated to poverty rates, rental prices and home values are correlated to income and often times, and neighborhoods in cities are often predominantly made up of a single race/ethnicity. With that being said, we found a correlation between these demographic features and internet adoption and availability.

Reviewer: ```suggestion
One of our most concerning findings is the relationship between mean neighborhood income and internet availability/adoption. Overall, a higher income in a neighborhood correlates with better internet availability and adoption. This trend continues when looking at race, housing and other demographic features. First, it is important to note the there are several demographic features that are closely correlated in society: number of children is correlated to poverty rates, rental prices and home values are correlated to income and often times, and neighborhoods in cities are often predominantly made up of a single race/ethnicity. With that being said, we found a correlation between these demographic features and internet adoption and availability.
```

> The trend seems to show that areas such as The Bronx, with the highest poverty rates and lowest income, have the lowest internet adoption and availability rates. This is contrasted with Manhattan, which has the highest income levels as well as the best internet coverage and adoption. The other demographic factors fall in line with these correlations as well.

Reviewer: ```suggestion
These trend show that areas such as the Bronx, with the highest poverty rates and lowest income, have the lowest internet adoption and availability rates. This is contrasted with Manhattan, which has the highest income levels as well as the best internet coverage and adoption. The other demographic factors fall in line with these correlations as well.
```

> Our findings show that the goal of Mayor de Blasio's Internet Master Plan (equitable internet access across New York City for everyone) has yet to be met. The progress of the plan may have been stifled by the COVID 19 pandemic, nonetheless, we can see that the "aggressive infrastructure bill" has failed to meet its goals thus far. 

Reviewer: ```suggestion
Our findings show that New York City has yet to meet Mayor de Blasio's Internet Master Plan goals of equitable internet access across the city.  While the COVID-19 pandemic may have stifled progress towards carrying out the plan, the "aggressive infrastructure bill" has failed to meet its goals thus far. 
```
